templates/vhost/vhost_ssl_header.erb

```diff
@@ -152,6 +152,12 @@ server {
 <% Array(@passenger_cgi_param).each do |key,value| -%>
   passenger_set_cgi_param  <%= key %> <%= value %>;
 <% end -%>
+<% Array(@passenger_header).each do |key,value| -%>
```

Comment:
Please call this parameter `passenger_set_header` so it matches the nginx directive name.